pgx-version-updater/src/main.rs

```diff
@@ -190,60 +190,42 @@ fn main() {
+                    current_version_specifier,
+                    &args.update_version.as_str(),
+                ))
+            }
+        };
+
         // Process dependencies in each file. Generally dependencies can be found in
         // [dependencies], [dependencies.foo], [build-dependencies], [dev-dependencies]
         for updatable_table_name in ["dependencies", "build-dependencies", "dev-dependencies"] {
             if let Some(updatable_table) =
                 doc.get_mut(updatable_table_name).and_then(|i| i.as_table_mut())
             {
                 for package in &updatable_package_names {
-                    match updatable_table.get_mut(package) {
-                        // Tables can contain other tables, and if that's the case we're
-                        // probably at a case of:
-                        //   [dependencies.pgx]
-                        //   version = "1.2.3"
-                        Some(item) if item.is_table() => {
-                            if let Some(current_version_specifier) =
-                                item.get("version").and_then(|a| a.as_str())
-                            {
-                                item["version"] = value(parse_new_version(
-                                    current_version_specifier,
-                                    &args.update_version.as_str(),
-                                ))
-                            }
-                        }
-
-                        // Inline table covers the case of:
-                        //   [dependencies]
-                        //   pgx = { version = "1.2.3", features = ["..."] }
-                        Some(item) if item.is_inline_table() => {
-                            if let Some(current_version_specifier) = item
-                                .as_inline_table()
-                                .and_then(|i| i.get("version"))
-                                .and_then(|v| v.as_str())
-                            {
-                                updatable_table[package]["version"] = value(parse_new_version(
-                                    current_version_specifier,
-                                    &args.update_version.as_str(),
-                                ))
-                            }
-                        }
-
-                        // Otherwise we are a string, such as:
-                        //   [dependencies]
-                        //   pgx = "0.1.2"
-                        Some(item) => {
-                            if let Some(current_version_specifier) = item.as_str() {
-                                updatable_table[package] = value(parse_new_version(
-                                    current_version_specifier,
-                                    &args.update_version.as_str(),
-                                ))
-                            }
-                        }
-
-                        // Don't care!
-                        None => {}
```

Comment:
So glad to get rid of this. It always bothered me, but didn't really know how to remove it or omit it without making it look weird. :)